packages/sdk/src/addresses/constants.ts

```diff
@@ -0,0 +1,12 @@
+import { invert } from "./helper";
+import type { AddressType, AddressFormat } from "./types";
+
+export const ADDRESS_TYPE_TO_FORMAT: Record<AddressType, AddressFormat> = {
+  p2pkh: "legacy",
+  p2sh: "nested-segwit",
```

Comment:
These names were used because of their names being represented in unisat's wallet.

![unisat-wallet-example](https://github.com/ordzaar/ordit-sdk/assets/10258208/bd44e54e-9dfd-4b5c-b275-0eb733fc3d63)

I actually find this variable confusing because it's not standardised, will it be an issue if we remove it and just use the address types natively instead of giving it names?

Reply:
Lets rename as suggested above and keep the variable for now. I am planning on refactoring Ordit class and native type names would be useful in that.